client/state/login/test/selectors.js

```diff
@@ -364,7 +369,7 @@ describe( 'selectors', () => {
 						socialAccountLink,
 					},
 				} )
-			).to.eql( true );
+			).toEqual( true );
```

Comment:
Nit: primitive values like numbers, booleans and strings can be compared with `.toBe`. `.toEqual` is usually used only to deep-compare arrays and objects.

Reply:
Replaced all those with `toBe` 👍